packages/fluentui/docs-components/src/ComponentControlsCodeSandbox/ComponentControlsCodeSandbox.tsx

```diff
@@ -25,7 +26,7 @@ type ComponentControlsCodeSandboxState = {
   sandboxUrl: string;
 };
 
-class ComponentControlsCodeSandbox extends React.Component<
+export class ComponentControlsCodeSandbox extends React.Component<
```

Comment:
```suggestion
export class CodeSandboxExporter extends React.Component<
```

Let's rename the component as now it's not a control in docs example 🐱